ddtrace/tracer/option.go

```diff
@@ -117,6 +118,22 @@ func WithHTTPRoundTripper(r http.RoundTripper) StartOption {
 	}
 }
 
+// WithAnalytics allows specifying whether Trace Search & Analytics should be enabled
+// for integrations.
+func WithAnalytics(on bool) StartOption {
+	if on {
+		return WithAnalyticsRate(1.0)
+	}
+	return WithAnalyticsRate(0.0)
+}
+
+// WithAnalyticsRate sets the global sampling rate for sampling APM events.
```

Comment:
Should this still refer to Trace Analytics events like other comments?

Reply:
I'm not sure. I think "APM events" is the official term (ref https://www.datadoghq.com/blog/trace-search-high-cardinality-data/)

Reply:
APM events are the official term 👍